fbpic/lpa_utils/bunch.py

```diff
@@ -345,11 +373,11 @@ def add_elec_bunch_openPMD( sim, ts_path, z_off=0., species=None, select=None,
     z = z - np.average(z, weights=w) + z_off
 
```

Comment:
In principle we could also use the charge and mass from the openPMD file here. I just wasn't sure if these properties were always saved and how backward compatible this is.